app/lib/account/backend.dart

```diff
@@ -562,6 +563,20 @@ class AccountBackend {
     }
   }
 
+  /// Retrieves a list of all uploader events that happened between [begin] and
+  /// [end].
+  Stream<AuditLogRecord> getUploadEvents({DateTime? begin, DateTime? end}) {
+    final query = _db.query<AuditLogRecord>();
+    query.filter('kind =', AuditLogRecordKind.packagePublished);
+    if (begin != null) {
+      query.filter('created >=', begin);
+    }
+    if (end != null) {
+      query.filter('created <', end);
+    }
+    return query.run();
+  }
```

Comment:
We'll probably need a new entry in index.yaml for this to work... so either we test in staging or we try to guess what that'll be...

Reply:
Tried on staging, and you were absolutely right.

Added an entry in index.yaml.